client/src/components/Alert.js

```diff
@@ -0,0 +1,45 @@
+import React, { useState, useEffect } from "react";
+import { Snackbar } from "@material-ui/core";
+import MuiAlert from "@material-ui/lab/Alert";
+
+function Alert(props) {
+  return <MuiAlert elevation={6} variant="filled" {...props} />;
+}
+
+function AlertMessage(props) {
+  const [open, setOpen] = useState(false);
+  const [message, setMessage] = useState("");
+
+  useEffect(() => {
+    console.log(props);
+    if (props.alert.error == true) {
```

Comment:
```suggestion
    if (props.alert.error) {
```
=== should be used in place of == but in this case you can just use the value itself